packages/scoutgame-ui/src/components/claim/components/PointsClaimScreen/PointsClaimModal/PointsClaimSocialShare.tsx

```diff
@@ -14,10 +17,20 @@ type ShareMessageProps = {
 
 export function PointsClaimSocialShare(props: Omit<ShareMessageProps, 'platform'>) {
   const isMd = useMdScreen();
+  const { refreshUser } = useUser();
+
+  const { execute, isExecuting } = useAction(completeQuestAction, {
+    onSuccess: () => {
+      refreshUser();
```

Comment:
are u sure we have to refresh the user profile when a quest is completed? seems odd